client/styles/index.css

```diff
@@ -0,0 +1,11 @@
+input[type=text]{
+    border: none;
+}
+
+#forms{
+    display: flex;
+    align-items: center; 
+    justify-content: center;
+    margin-top: 10% ;
+}
+    
```

Comment:
Maybe a bit more thought could have been put into the styling of the page, but I do appreciate there was a time limitation as well.